megalista_dataflow/main.py

```diff
@@ -44,12 +46,15 @@
 from uploaders.google_analytics.google_analytics_measurement_protocol import \
   GoogleAnalyticsMeasurementProtocolUploaderDoFn
 from uploaders.google_analytics.google_analytics_user_list_uploader import GoogleAnalyticsUserListUploaderDoFn
+from uploaders.display_video.customer_match.contact_info_uploader import DisplayVideoCustomerMatchContactInfoUploaderDoFn
+from uploaders.display_video.customer_match.mobile_uploader import DisplayVideoCustomerMatchMobileUploaderDoFn
 
 warnings.filterwarnings(
     "ignore", "Your application has authenticated using end user credentials"
 )
 
 ADS_CM_HASHER = AdsUserListPIIHashingMapper()
+DV_CM_HASHER = DVUserListPIIHashingMapper()
```

Comment:
The main difference was the name of the keys, for example  ads uses "hashed_first_name" while dv takes "hashedFirstName", and the composition of the contact information. For example,  ads includes the contact information fields nested under "address_info" while dv does not. 

I've created an abstract hashing mapper class in order to avoid some of the repetition.